pandas/tests/frame/test_arithmetic.py

```diff
@@ -138,3 +143,252 @@ def test_ops_frame_period(self):
+                                 for k, v in tm.getSeriesData().items()},
+                                dtype=np.int64)
+        mixed_int = pd.DataFrame({'A': intframe['A'].copy().astype('int32'),
+                                  'B': np.ones(len(intframe['B']),
+                                               dtype='uint64'),
+                                  'C': intframe['C'].copy().astype('uint8'),
+                                  'D': intframe['D'].copy().astype('int64')})
+
+        dunder_func = getattr(operator, opname)
+
+        # vs mix int
+        result = getattr(mixed_int, opname)(2 + mixed_int)
+        expected = dunder_func(mixed_int, 2 + mixed_int)
+        tm.assert_frame_equal(result, expected)
+
+        dtype = {'C': None}
+        if opname == 'sub':
+            dtype['B'] = 'uint64'
+
+        # no overflow in the uint
+        _check_mixed_int(result, dtype=dtype)
+
+        # vs mix float
+        result = getattr(mixed_float, opname)(2 * mixed_float)
+        expected = dunder_func(mixed_float, 2 * mixed_float)
+        tm.assert_frame_equal(result, expected)
+        _check_mixed_float(result, dtype={'C': None})
+
+        result = getattr(intframe, opname)(2 * intframe)
+        expected = dunder_func(intframe, 2 * intframe)
+        tm.assert_frame_equal(result, expected)
+
+        # vs mix int
+        result = getattr(mixed_int, opname)(2 + mixed_int)
+        expected = dunder_func(mixed_int, 2 + mixed_int)
+        tm.assert_frame_equal(result, expected)
+
+        # no overflow in the uint
+        _check_mixed_int(result, dtype=dtype)
+
+    @pytest.mark.parametrize('opname', ['add', 'sub', 'mul',
+                                        'div', 'truediv',
+                                        'pow', 'floordiv', 'mod'])
+    def test_df_arith_flex_float_dtypes_match_dunder(self, opname):
+        # TODO: Is this supposed to be about the dtypes or about the
+        # indexed_same?
+        frame = pd.DataFrame(tm.getSeriesData())
+
+        mixed_float = pd.DataFrame({'A': frame['A'].copy().astype('float32'),
+                                    'B': frame['B'].copy().astype('float32'),
+                                    'C': frame['C'].copy().astype('float16'),
+                                    'D': frame['D'].copy().astype('float64')})
+
+        alias = 'truediv' if PY3 and opname == 'div' else opname
+        try:
+            dunder_func = getattr(operator, alias)
+
+            # vs mix float
+            result = getattr(mixed_float, opname)(2 * mixed_float)
+            expected = dunder_func(mixed_float, 2 * mixed_float)
+            tm.assert_frame_equal(result, expected)
+            _check_mixed_float(result, dtype={'C': None})
+
+        except:
+            import pandas.io.formats.printing as printing
+            printing.pprint_thing("Failing operation %r" % opname)
+            raise
+
+    def test_df_arith_flex(self):
+        pass
```

Comment:
What happened here?

Reply:
These tests are being split from (see the diff of `test_operators`) `test_arith_flex_frame`, which has these lines commented-out in the status quo.  Retained them on the "they're here for a reason, I just don't know it yet" principle.